### PR TITLE
building: qemu: add fetch/build/run commands example

### PR DESCRIPTION
**Pull request by etienne-lms**
Add an explicit example of shell commands to get, build and run
OP-TEE on Qemu (v7 and v8). This is quite helpful for one reading
the documentation to easy copy/paste required commands, rather than
digging back in generic build instruction to setup the right
command series.

Signed-off-by: Etienne Carriere <etienne.carriere@linaro.org>

### CODE REVIEW

File: building/devices/qemu.rst [hunk 1]
```diff
@@ -14,6 +14,16 @@ As long as you pick the v7 manifest, i.e.,  ``default.xml`` the
 ":ref:`get_and_build_the_solution`" tells all you need to know to build and boot
 up QEMU v7.
 
+A usual short shell sequence to fecth, build and run OP-TEE using Qemu
```
Comment (jforissier) [inline building/devices/qemu.rst:17]: fetch, QEMU

File: building/devices/qemu.rst [hunk 2]
```diff
@@ -218,6 +228,17 @@ As long as you pick the v8 manifest, i.e.,  ``qemu_v8.xml`` the
 ":ref:`get_and_build_the_solution`" tells all you need to know to build and boot
 up QEMU v8.
 
+A usual short shell sequence to fecth, build and run OP-TEE usnig Qemu
```
Comment (jforissier) [inline building/devices/qemu.rst:231]: fetch, using, QEMU

**Review by jforissier — COMMENTED** [review]
Should the FAQ entry (https://optee.readthedocs.io/en/latest/faq/faq.html#q-what-is-the-quickest-and-easiest-way-to-try-op-tee) just link to one of the two new paragraphs then?

`Reviewed-by: Jerome Forissier <jerome@forissier.org>`

**Comment by etienne-lms** [comment]
I would keep the FAQ as-is, despite it duplicates the info. Qemu is quite specific, maintained by OP-TEE maintainers so the commands will unlikely change. Better to one reading to FAQ gets straight the commands cheatsheet, rather than a link, specifically the FAQ covers include prerequisite and consoles stuff.

**Comment by jforissier** [comment]
@etienne-lms OK fine with me.

**Review by jbech-linaro — COMMENTED** [review]
With the spelling errors fixed:
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Comment by etienne-lms** [comment]
typos fixed, tags applied.
tahnks.

**Pull request merged**